events/blocks/events-form/events-form.js

```diff
@@ -565,6 +576,8 @@ export default async function decorate(block, formData = null) {
     form: block.querySelector(':scope > div:nth-of-type(2) a[href$=".json"]'),
     terms: block.querySelector(':scope > div:nth-of-type(3)'),
     successMsg: block.querySelector(':scope > div:last-of-type > div'),
+    title: block.querySelector(':scope #event-title'),
+    modal: block.closest('.dialog-modal'),
```

Comment:
You also might not need to keep this separately in the bp (blueprint) because you can access it in the `eventFormSendAnalytics` by doing the same `bp.block.closest('.dialog-modal')`. The only reason we querySelector here and keep the other elements in the bp is because the block DOM structure changes in the process of decoration and if we don't keep track of them here it becomes a lot more unpredictable in later steps. I could be wrong but that shouldn't be the case for getting the closest `.dialog-modal`.